tests/test.py

```diff
@@ -307,6 +307,9 @@ def _test_indentation(self, filename, contents):
     def _test_package(self, include, data):
         name = get_package_name(data)
 
+        with _open("repository/labels.json") as labels_file:
+            labels_list = json.loads(labels_file.read().decode("utf-8", "strict"), encoding="utf-8")
```

Comment:
Doing IO for every package test is not a good idea. If we are indeed going forward with this, please cache the contents of the labels file somewhere.
Also, I think a simple textfile with the labels on their own line is simpler and works just as well.